budget/analytic.py

```diff
@@ -29,3 +29,13 @@ class account_analytic_account(orm.Model):
                                            'analytic_account_id',
                                            string='Budget Lines'),
     }
+
+    def copy_data(self, cr, uid, id, default=None, context=None):
+        """Remove budget lines when copying analytic account"""
+        if default is None:
+            default = {}
+        else:
+            default = default.copy()
+        default['budget_line_ids'] = False
```

Comment:
It's better to use
```
default.setdefault('budget_line_ids', False)
```
So if a `'budget_line_ids'` key is already set in the dict, it will be preserved.